views/index-green.ejs

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>SOBIE 2026 - Home</title>
+
+  <!-- Google Font Link -->
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;500;700&display=swap" rel="stylesheet">
+
+  <!-- External CSS Link -->
+  <link rel="stylesheet" href="style-green.css" />
+</head>
+<body>
+  <!-- Sidebar Navigation -->
+  <nav class="sidebar">
+    <h2>SOBIE 2026</h2>
+    <ul>
+      <li><a href="index-green.ejs" class="active">Home</a></li>
```

Comment:
all of these need to be endpoints vs. links to `.ejs` => `index-green/`

with either 

a) a corresponding endpoint in Home Controllers 
or
b) info in the PR to create the endpoints for us to help you with.